File: Linguagem C#/Repetitiva/ParImpar.cs
using System;

namespace ParImpar {
    class Program {
        static void Main(string[] args) {

            int n, x;

            Console.Write("Quantos numeros voce vai digitar? ");
            n = int.Parse(Console.ReadLine());

            for (int i = 1; i <= n; i++) {
                Console.Write("Digite um numero: ");
                x = int.Parse(Console.ReadLine());

                if (x % 2 == 0 && x > 0) {
                    Console.WriteLine("PAR POSITIVO");
                }else if(x % 2 ==0 && x < 0) {
                    Console.WriteLine("PAR NEGATIVO");
                }else if(x % 2 != 0 && x > 0) {
                    Console.WriteLine("IMPAR POSITIVO");
                }else if (x % 2 != 0 && x < 0) {
                    Console.WriteLine("IMPAR NEGATIVO");
                }else {
                    Console.WriteLine("NULO");
                };
            };
        }
    }
}

File: Linguagem Python/Repetitiva/ParImpar.py
n = int(input("Quantos casos voce vai digitar? "))

for i in range (1,n+1):
    print("Digite tres numeros:")
    n1 = float(input())
    n2 = float(input())
    n3 = float(input())
    media = (n1*2+n2*3+n3*5) / 10
    print(f"MEDIA = {media:.1f}")